toolkit/docs/building/building.md

```diff
@@ -509,6 +509,16 @@ If that is not desired all remote sources can be disabled by clearing the follow
 
 > Delete all `marinertoolchain*` containers and images associated with this working directory when running `make clean`.
 
+#### `DELTA_FETCH=...`
```

Comment:
I think we should also update the table of contents at the very top.

Reply:
This PR completely removes the table of contents (we are recommending people use the built in github TC. Unfortunately there isn't a way to make one automatically in markdown).

https://github.com/microsoft/CBL-Mariner/pull/5218/files#diff-1992872e23ca4d1d6b1e714885792b2150b6c7f266e981c7de90d71c6fad388f